src/Ocelot/LoadBalancer/LoadBalancers/IStickySessionStorage.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Ocelot.LoadBalancer.LoadBalancers;
+
+public interface IStickySessionStorage
```

Comment:
Well... In PR you said:
> - make a public interface for session storage so users can implement their own mechanism of storing user sessions such as distributed sessions storage etc.

Based on the current code I can say, the interface object is not added to DI-container.

Reply:
Still the issue!